src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -129,29 +130,61 @@ public void OnResponseHeader(ReadOnlySpan<byte> name, ReadOnlySpan<byte> value)
                         throw new Http2ProtocolException(Http2ProtocolErrorCode.ProtocolError);
                     }
 
-                    if (name.SequenceEqual(s_statusHeaderName))
+                    if (name[0] == 58) // ':'
```

Comment:
`58` => `(byte)':'`

Reply:
Also, I assume it's not possible for `name` to be empty?  Should we assert that at the beginning of the method?